# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../heimdal/glibc-2.5-getnameinfo.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- heimdal-0.8-rc1/lib/roken/getnameinfo.c.orig	2006-11-07 15:09:06.000000000 +0200
+++ heimdal-0.8-rc1/lib/roken/getnameinfo.c	2006-11-07 15:16:02.000000000 +0200
@@ -94,6 +94,7 @@
  *
  */
 
+#ifndef HAVE_GETNAMEINFO
 int ROKEN_LIB_FUNCTION
 getnameinfo(const struct sockaddr *sa, socklen_t salen,
 	    char *host, size_t hostlen,
@@ -125,3 +126,4 @@
 	return EAI_FAMILY;
     }
 }
+#endif
